tests/main/windows-spec.ts

```diff
@@ -92,7 +92,8 @@ describe('windows', () => {
       expect(createContextMenu).toHaveBeenCalled();
     });
 
-    it('prevents new-window"', () => {
+    // FIXME: new test for setWindowOpenHandler
+    it.skip('prevents new-window"', () => {
```

Comment:
Not sure how to directly test this tbh :/

Reply:
Might need to take inspiration from the Electron unit tests: https://github.com/electron/electron/blob/81793496258b20c6134ea14843cbc91a0315d566/spec-main/guest-window-manager-spec.ts#L94